x/cronos/keeper/evm_log_handlers.go

```diff
@@ -30,11 +32,16 @@ var (
 	// EthereumTransferEvent represent the signature of
 	// `event __CronosSendToEthereum(address recipient, uint256 amount, uint256 bridge_fee)`
 	EthereumTransferEvent abi.Event
+
+	// IbcTransferEvent represent the signature of
+	// `event __CronosSendToIbc(string recipient, uint256 amount)`
+	IbcTransferEvent abi.Event
```

Comment:
Shall we rename these variables based on the event name? i.e. `SendToIbcEvent`

I sometimes find it confusing when I look at `IbcTransferEvent` in the code